drivers/include/periph/rtt.h

```diff
@@ -79,24 +79,30 @@ extern "C" {
 #define RTT_MIN_OFFSET (2U)
 #endif
 
-#ifndef RTT_FREQUENCY
 /* Allow mock-RTT for unit tests */
 #ifdef MOCK_RTT_FREQUENCY
-#define RTT_FREQUENCY MOCK_RTT_FREQUENCY
+  #ifdef RTT_FREQUENCY
+    #undef RTT_FREQUENCY
+  #endif
```

Comment:
You can unconditionally `#undef` the macro, keeps the `#ifdef` block a bit slimmer.

Reply:
Applied